core/message-tags-3.3.md

```diff
@@ -135,9 +135,14 @@ Servers MUST reply with the `ERR_INPUTTOOLONG` (`417`) error numeric if a client
 ## Security considerations
 
 Client-only tags should be treated as untrusted data. They can contain any value
-and are not validated by servers in any way.
+and are not validated by servers in any way. The server MAY normalise values by applying the unescape
+and escape process.
```

Comment:
Also normalization implies there's more than one way to do escaping, but the escapes specified in the spec are so limited that there's not much room to do that, if any. You would need to consider invalid inputs. So:

Less escapes than needed break parsing always, there's no reasonable way to recover or detect it. Unescaped semicolon ends the value, unescaped space ends the tag, unescaped CR/LF end the message.

Unknown escapes are a more interesting edge case. Does `escape(unescape("\a"))` turn into `a`, `\a` or `\\a`? Out of scope of this PR btw, just saying that it probably deserves more than an offhand "server MAY clean the mess".